src/Gibe.AbTest/Dates/DateTimeUtcTimeProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+// ReSharper disable DTN
+
+namespace Gibe.AbTest.Dates
+{
+	public class DateTimeUtcTimeProvider : ITimeProvider
```

Comment:
Where does this get bound to DI?  We would normally have some kind of NinjectModule, but with this being shared between netcore & framework there's two different ways of doing it.

Should we have some kind of helper method to help register all of the DI stuff as we do for Freshdesk which can be called in the site DI setup?

https://github.com/Gibe/BeyondRetail/blob/develop/src/TapWarehouse.Freshdesk/Services.cs